docs/usage/docker.md

```diff
@@ -62,59 +66,62 @@ You can configure that with another package rule:
-Read on to see how Renovate keeps it up-to-date.
+To keep things human-friendly, Renovate will retain the Docker tag in the `FROM` line, e.g. `FROM node:14.15.1@sha256:d938c1761e3afbae9242848ffbb95b9cc1cb0a24d889f8bd955204d347a7266e`.
+Read on to see how Renovate updates Docker digests.
 
 ## Digest Updating
 
-If you have followed our advice to go from tags like `node:8` to `node:8@sha256:552348163f074034ae75643c01e0ba301af936a898d778bb4fc16062917d0430`, then you are likely to receive Renovate PRs whenever the `node:8` image is updated on Docker Hub.
+If you follow our advice to go from a simple tag like `node:14` to using a pinned digest `node:14@sha256:d938c1761e3afbae9242848ffbb95b9cc1cb0a24d889f8bd955204d347a7266e`, you will receive Renovate PRs whenever the `node:14` image is updated on Docker Hub.
 
-Previously this would have been "invisible" to you - one day you pull code that represents `node:8.9.0` and the next day you get `node:8.9.1`.
+Previously this update would have been "invisible" to you - one day you pull code that represents `node:14.9.0` and the next day you get code that represents `node:14.9.1`.
 But you can never be sure, especially as Docker caches.
-Perhaps some of your colleagues or worst still your build machine are stuck on an older version with a security vulnerability.
+Perhaps some of your colleagues or worse still your build machine are stuck on an older version with a security vulnerability.
 
-Instead, you will now receive these updates via Pull Requests, or perhaps committed directly to your repository if you enable branch automerge for convenience.
-This ensures everyone on the team gets the latest versions and is in sync.
+By pinning to a digest instead, you will receive these updates via Pull Requests, or even committed directly to your repository if you enable branch automerge for convenience.
+This ensures everyone on the team uses the latest versions and is in sync.
 
 ## Version Upgrading
 
-Renovate also supports _upgrading_ versions in Docker tags, e.g. from `node:8.9.0` to `node:8.9.1` or `node:8.9` to `node:8.10`.
+Renovate also supports _upgrading_ versions in Docker tags, e.g. from `node:14.9.0` to `node:14.9.1` or `node:14.9` to `node:14.10`.
 If your tags looks like a version, Renovate will upgrade it like a version.
 
-Thanks to this, you may wish to change the way you tag your image dependencies to be more specific, e.g. change from `node:8` to `node:8.9.1` so that every Renovate PR will be more human friendly, e.g. you can know that you are getting a PR because `node` upgraded from `8.9.1` to `8.9.2` and not because `8.9.1` somehow changed.
+Thanks to this, you may wish to change the way you tag your image dependencies to be more specific, e.g. change from `node:14` to `node:14.9.1` so that every Renovate PR will be more human friendly, e.g. you can know that you are getting a PR because `node` upgraded from `14.9.1` to `14.9.2` and not because `14.9.1` somehow changed.
```

Comment:
> Thanks to this, you may wish to change the way you tag your image dependencies to be more specific, e.g. change from `node:14` to `node:14.9.1` so that every Renovate PR will be more human friendly, e.g. you can know that you are getting a PR because `node` upgraded from `14.9.1` to `14.9.2` and not because `14.9.1` somehow changed.

This sentence is still hard to understand.
I think what we're trying to say here is:

"Use a more specific tag like `node:14.9.1` so that you can tell if you're getting a update for `node:14.9.2` or because `node:14.9.1` changed."

I don't know what the right sentence would be, I have tried rewording this, but I can't get a nice clear sentence out of it. Do you have some ideas?